index.bs

```diff
@@ -2542,6 +2642,63 @@ To <dfn>obtain and deliver a debug report on source registration</dfn> given a
 1. Run [=obtain and deliver a debug report=] with « |data| », |source|'s [=attribution source/reporting origin=],
     and |source|'s [=attribution source/fenced=].
 
+To <dfn>find sources with common destinations and reporting origin</dfn> given an [=attribution source=] |pendingSource|:
+1. [=Assert=]: |pendingSource| is in the [=attribution source cache=].
+1. Let |cache| be the user agent's [=attribution source cache=].
```

Comment:
This variable is used only 1 time. Maybe just inline it?

Reply:
Nice catch, done!